.circleci/config.yml

```diff
@@ -1,74 +1,118 @@
 aliases:
-  - &Determine-Tests-to-Run
+  - &setup-env
+    name: Setup Envionrment
+    shell: /bin/bash
+    command: source .circleci/ci-helper.sh && envSetup
+
+  - &determine-tests-to-run
     name: Determine Tests to Run
     shell: /bin/bash
     command: source .circleci/ci-helper.sh && printTestsToRun
 
-  - &Run-Danger
+  - &run-danger
     name:  Run Danger
     when: always
     command:  source .circleci/ci-helper.sh && runDanger
 
-  - &Launch-Emulator
+  - &launch-emulator
     name:  Launch Emulator
     command: source .circleci/ci-helper.sh && startAVD
     background: true
 
-  - &Wait-for-Emulator
+  - &wait-for-emulator
     name:  Wait for Emulator to boot
     command: source .circleci/ci-helper.sh && waitForAVD
 
-  - &Lib-Run-Base
+  - &lib-run-base
     halt_build_on_fail: false
     no_output_timeout: 900
 
-  - &Run-Unit-Tests-PR
-    <<: *Lib-Run-Base
+  - &run-unit-tests
+    <<: *lib-run-base
     command: source .circleci/ci-helper.sh && runTests
 
-  - &Run-SalesforceAnalytics
-    <<: *Lib-Run-Base
-    name: Run SalesforceAnalytics
+  - &test-salesforce-analytics
+    <<: *lib-run-base
+    name: Test SalesforceAnalytics
     command: ./gradlew :libs:SalesforceAnalytics:connectedAndroidTest --continue --no-daemon --profile --max-workers 2
     when: always
 
-  - &Run-SalesforceSDK
-    <<: *Lib-Run-Base
-    name: Run SalesforceSDK
+  - &test-salesforce-sdk
+    <<: *lib-run-base
+    name: Test SalesforceSDK
     command: ./gradlew :libs:SalesforceSDK:connectedAndroidTest --continue --no-daemon --profile --max-workers 2
     when: always
 
-  - &Run-SmartStore
-    <<: *Lib-Run-Base
-    name: Run SmartStore
+  - &test-smart-store
+    <<: *lib-run-base
+    name: Test SmartStore
     command: ./gradlew :libs:SmartStore:connectedAndroidTest --continue --no-daemon --profile --max-workers 2
     when: always
 
-  - &Run-SmartSync
-    <<: *Lib-Run-Base
-    name: Run SmartSync
+  - &test-smart-sync
+    <<: *lib-run-base
+    name: Test SmartSync
     command: ./gradlew :libs:SmartSync:connectedAndroidTest --continue --no-daemon --profile --max-workers 2
     when: always
 
-  - &Run-SalesforceHybrid
-    <<: *Lib-Run-Base
-    name: Run SalesforceHybird
+  - &test-salesforce-hybrid
+    <<: *lib-run-base
+    name: Test SalesforceHybird
     command: ./gradlew :libs:SalesforceHybrid:connectedAndroidTest --continue --no-daemon --profile --max-workers 2
     when: always
 
-  - &Build-SalesforceReact
-    <<: *Lib-Run-Base
-    name: Run SalesforceReact
+  - &test-salesforce-react
+    <<: *lib-run-base
+    name: Build SalesforceReact
     command: |
-      ./gradlew :libs:SalesforceReact:assembleDebug -p . --continue --no-daemon --profile --max-workers 2
-      ./gradlew :libs:SalesforceReact:assembleRelease -p . --continue --no-daemon --profile --max-workers 2
+      ./gradlew :libs:SalesforceReact:assemble
```

Comment:
`assembleDebug`?

Reply:
`assemble` by itself runs `assembleDebug` and `assembleRelease` right?

Reply:
Correct (http://tools.android.com/tech-docs/new-build-system/user-guide#TOC-Android-tasks)